static/bidder-info/infytv.yaml

```diff
@@ -0,0 +1,9 @@
+maintainer:
+  email: "tech+hb@infy.tv"
+capabilities:
```

Comment:
I would like to confirm InfyTV does not have an IAB Europe GVL vendor id.

Reply:
@SyntaxNode Yes, InfyTV does not have an IAB Europe GVL vendor id.